frontend/src/services/fetch/fetchers/agenciesFetcher.ts

```diff
@@ -41,62 +39,62 @@ export const obtainAgencies = async (): Promise<RelevantAgencyRecord[]> => {
   return data;
 };
 
-// translates API response containing flat list of agencies into nested filter options
-export const agenciesToFilterOptions = (
```

Comment:
diff is tough here - this, and other non-fetch related functionality, was moved to searchUtils